### ISSUE TITLE
Cannot permanently disable vizion

### ISSUE THREAD

**Issue by pmalouin** [Unitech/pm2#3105]
## What's going wrong?
I have very slow performance of git commands on the environment on which I work. When I run a `pm2 start all` command, I can see a handful of "git status" commands using up a lot of CPU (I see those commands with the `htop` tool). I don't really understand why pm2 tries to analyze the git repo, but I actually expect the git repo not to be valid (I'm file-sharing the repo from a windows host to a guest debian VM).

Trying to work around the issue, I want to use the --no-vizion option or the "vizion" attribute set to `false` in the process file. The docs say that this setting will disable all git commands.

With the following process.json file (shortened to only one process):
```
{
  "apps": [
    {
      "name": "router",
      "script": "router/app.js",
      "vizion": false,
      "env": {
        "NODE_ENV": "development"
      },
      "env_production": {
        "NODE_ENV": "production"
      }
    }]
}
```
Running `pm2 start process.json`, I can see that there are no "git status" commands spawned anymore. Unfortunately, if I restart the process using `pm2 start all`, I can see "git status" commands again in `htop`. It seems the configuration value is not remembered across restarts.

I also tried the command `pm2 start all --no-vizion`, but again, I can still see "git status" commands.

## How could we reproduce this issue?
Since the git commands on my system use a lot of CPU and run for many seconds, I can see the git processes in the `htop` tool.

## Supporting information
```
===============================================================================
--- PM2 REPORT (Mon Aug 21 2017 15:18:07 GMT-0400 (EDT)) ----------------------
===============================================================================
--- Daemon -------------------------------------------------
pm2d version         : 2.6.1
node version         : 6.11.0
node path            : /home/vagrant/.npm-packages/bin/pm2
argv                 : /usr/bin/nodejs,/home/vagrant/.npm-packages/lib/node_modules/pm2/lib/Daemon.js
argv0                : node
user                 : vagrant
uid                  : 1000
gid                  : 1000
uptime               : 91min
===============================================================================
--- CLI ----------------------------------------------------
local pm2            : 2.6.1
node version         : 6.11.0
node path            : /home/vagrant/.npm-packages/bin/pm2
argv                 : /usr/bin/nodejs,/home/vagrant/.npm-packages/bin/pm2,report
argv0                : node
user                 : vagrant
uid                  : 1000
gid                  : 1000
===============================================================================
--- System info --------------------------------------------
arch                 : x64
platform             : linux
type                 : Linux
cpus                 : Intel(R) Core(TM) i7-5500U CPU @ 2.40GHz
cpus nb              : 2
freemem              : 2991685632
totalmem             : 4156284928
home                 : /home/vagrant
===============================================================================
```

I can see some earlier issues with some similar problems (#2867, #1074)

**Comment by Unitech** [comment]
fixed via https://github.com/Unitech/pm2/issues/3085

**Comment by Unitech** [comment]
Fixed on recently published pm2 (2.7.0):

```bash
$ npm install pm2@latest -g
$ pm2 update
```

Let me know if there is any issue
